### PR TITLE
Update to IV 4.x

### PR DESCRIPTION
**Pull request by dwieeb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,11 +19,11 @@
...
-    "@capacitor/cli": "~1.5.0",
-    "@capacitor/core": "~1.5.0",
-    "@capacitor/ios": "~1.5.0",
-    "@ionic-enterprise/identity-vault": "~3.6.1",
+    "@capacitor/android": "^2.0.0",
+    "@capacitor/cli": "^2.0.0",
+    "@capacitor/core": "^2.0.0",
+    "@capacitor/ios": "^2.0.0",
+    "@ionic-enterprise/identity-vault": "^4.0.0-1",
     "@ionic/angular": "~5.0.1",
```
Comment (kensodemann) [inline package.json:27]: I generally prefer that `~` be used.

Comment (dwieeb) [reply package.json:27]: you're too fast, I'm still resolving merge conflicts

Comment (kensodemann) [reply package.json]: That said, latest `master` already has Cap 2.0 installed. I just hadn't finished updating it (I still needed to migrate the platforms as per the Capacitor docs. So it looks like this branch probably needs to be rebased.

**Comment by dwieeb** [comment]
@kensodemann this PR is ready for review!

**Pull request merged**